### DISCUSSION TITLE
Playtime stop setting, enhancement

### DISCUSSION THREAD

**Discussion by JonGraef** [AntonyCorbett/OnlyM#406] [Ideas]
Hi Anthony,
I was just looking at presenting the Sunday talk from the JW Stream download file. I have set the start time, but wondered if you could add an end time setting.
Jon Graef Bromsgrove Cong.

**Comment by JonGraef** [comment]
Is there any progress on adding this feature?
thanks

**Comment by AntonyCorbett** [comment]
@JonGraef Hi. Jon. There are no definite plans to add this. Unless an idea/feature request has support from many I can't really consider it.

**Reply by JonGraef** [reply]
I understand. I saw there was a start time setting and thought it might be straightforward to add a stop time.

**Comment by AntonyCorbett** [comment]
@JonGraef Yes, thanks. It may well be straightforward and sometimes I pick up the low-hanging fruit :)
